torchquantum/algorithm/__init__.py

```diff
@@ -25,3 +25,4 @@
 from .vqe import *
 from .hamiltonian import *
 from .qft import *
+from .grover import *
```

Comment:
Main fix is just adding this import; it must’ve gotten lost somehow when merging everything.